### DISCUSSION TITLE
Whisper cannot translate English to Italian

### DISCUSSION THREAD

**Discussion by pdhd97** [openai/whisper#2225] [Q&A]
I want to translate my audio file into Italian, but it shows the following result. What does it mean? Can I use Whisper to translate into Italian?

Command: !whisper "audio.mp3" --model medium --language it --task transcribe 
Result: Sottotitoli creati dalla comunità Amara.org."

The result "Sottotitoli creati dalla comunità Amara.org" means "Subtitles created by the Amara.org community" in English.

**Comment by glangford** [comment]
> Can I use Whisper to translate into Italian?

Whisper will not reliably translate into non-English languages.

See

* https://github.com/openai/whisper/discussions/649#discussioncomment-4325122
* https://github.com/openai/whisper/discussions/2105#discussioncomment-8900004
* https://github.com/openai/whisper/discussions/2009#discussioncomment-8410035

**Reply by pdhd97** [reply]
But when i translate into French. it works fine.

**Reply by glangford** [reply]
Translating to non-English is unpredictable. It sometimes works for Portuguese for example, but not always.
